scalardb-analytics-postgresql-sample/README.md

```diff
@@ -0,0 +1,233 @@
+| c_mktsegment | TEXT   | false        | false          | false           | null            | false     |
+| c_comment    | TEXT   | false        | false          | false           | null            | false     |
++--------------+--------+--------------+----------------+-----------------+-----------------+-----------+
+```
+
+You have the foreign table of `dynamons._customer` in the PostgreSQL database as well.
+
+```shell
+$ psql -U postgres -h localhost test -c '\d dynamons._customer';
+Password for user postgres:
+                            Foreign table "dynamons._customer"
+         Column         |       Type       | Collation | Nullable | Default | FDW options
+------------------------+------------------+-----------+----------+---------+-------------
+ c_custkey              | integer          |           |          |         |
+ c_name                 | text             |           |          |         |
+ c_address              | text             |           |          |         |
+ c_nationkey            | integer          |           |          |         |
+ c_phone                | text             |           |          |         |
+ c_acctbal              | double precision |           |          |         |
+ c_mktsegment           | text             |           |          |         |
+ c_comment              | text             |           |          |         |
+ tx_id                  | text             |           |          |         |
+ tx_version             | integer          |           |          |         |
+ tx_state               | integer          |           |          |         |
+ tx_prepared_at         | bigint           |           |          |         |
+ tx_committed_at        | bigint           |           |          |         |
+ before_tx_id           | text             |           |          |         |
+ before_tx_version      | integer          |           |          |         |
+ before_tx_state        | integer          |           |          |         |
+ before_tx_prepared_at  | bigint           |           |          |         |
+ before_tx_committed_at | bigint           |           |          |         |
+ before_c_name          | text             |           |          |         |
+ before_c_address       | text             |           |          |         |
+ before_c_nationkey     | integer          |           |          |         |
+ before_c_phone         | text             |           |          |         |
+ before_c_acctbal       | double precision |           |          |         |
+ before_c_mktsegment    | text             |           |          |         |
+ before_c_comment       | text             |           |          |         |
+Server: multi_storage_dynamodb
+FDW options: (namespace 'dynamons', table_name 'customer')
+```
+
+As you can see, the tables contains the transaction metadata columns as well. This is required to ensure the Read-Committed isolation level.
```

Comment:
In this case, I assumed `This` at the beginning of the second sentence meant `These columns`. But I wonder if it should be `This table` instead. Can you please confirm which one is accurate?

```suggestion
As you can see in the foreign table, the table contains the transaction metadata columns as well. These columns are required to ensure the Read Committed isolation level.
```